src/main/java/com/thealgorithms/others/Sudoku.java

```diff
@@ -86,22 +105,36 @@ public static boolean solveSudoku(int[][] board, int n) {
         return false;
     }
 
+    /**
+     * Prints the current state of the Sudoku board in a readable format.
+     * Each row is printed on a new line, with numbers separated by spaces.
+     *
+     * @param board The current state of the Sudoku board.
+     * @param n     The size of the Sudoku board (typically 9 for a standard puzzle).
+     */
     public static void print(int[][] board, int n) {
-        // We got the answer, just print it
+        // Print the board in a 9x9 grid format
```

Comment:
Same question here also.

Reply:
Changed this comment to `Print the board in a nxn grid format`